SpatialGDKEditorToolbar/Source/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -138,16 +138,58 @@ void WriteSchemaRPCField(TSharedPtr<FCodeWriter> Writer, const TSharedPtr<FUnrea
 	);
 }
 
+// core_types.schema should only be included if any components in the file have
+// 1. An UnrealObjectRef
+// 2. A list of UnrealObjectRefs
+// 3. A RPC
+bool ShouldIncludeCoreTypes(TSharedPtr<FUnrealType>& TypeInfo)
+{
+	FUnrealFlatRepData RepData = GetFlatRepData(TypeInfo);
+
+	for (auto& PropertyGroup : RepData)
+	{
+		for (auto& PropertyPair : PropertyGroup.Value)
+		{
+			UProperty* Property = PropertyPair.Value->Property;
+			if(Property->IsA(UObjectPropertyBase::StaticClass()))
```

Comment:
Nit: space.
Also, personal preference for `Property->IsA<UObjectPropertyBase>()`